include/opcua.h

```diff
@@ -108,15 +109,17 @@ class OPCUA
 	SOPC_ClientHelper_GetEndpointsResult
 				*GetEndPoints(const char *endPointUrl);
 	std::string		securityMode(OpcUa_MessageSecurityMode mode);
-	std::string		nodeClass(OpcUa_NodeClass nodeClass);
+	std::string		nodeClassStr(OpcUa_NodeClass nodeClass);
 	void			resolveDuplicateBrowseNames();
+	bool 			checkFiltering(const std::string& browseName, OpcUa_NodeClass nodeClass, bool isDirectlySubscribed=false);
+	
 	// void			getParents();
 	int32_t			m_connectionId;
 	int32_t			m_configurationId;
         std::vector<std::string>
 				m_subscriptions;	// The user subscriptions
 	std::map<std::string, Node *>
-				m_nodes;		// The nodes being monitored
+				m_nodes;		// The node information cache
```

Comment:
m_nodes has a very particular meaning, it is the nodes that we are currently expecting to receive data for and is used in a lookup in the callback. I think changing this comment is misleading.

Reply:
ok